spiceaidocs/docs/components/data-connectors/mysql.md

```diff
@@ -22,21 +22,94 @@ datasets:
 
 ## Configuration
 
+### `from`
+
+The `from` field takes the form `mysql:path.to.my_dataset` where `path.to.my_dataset` is the fully-qualified table name in the SQL server.
```

Comment:
MySQL only supports a two-level path scheme, of the format `database_name.table_name`. So specifying a three part path here is incorrect. This documentation of `from` should explain how the logical grouping model of MySQL maps to the path selector. i.e. db.table or just table for the default db

Reply:
It's interesting because we also support `mysql_db` as a parameter

Reply:
Fixed!